### ISSUE TITLE
Custom Checkbox - Cannot read Property Id of undefined

### ISSUE THREAD

**Issue by Ajaybhardwaj7** [AllenFang/react-bootstrap-table#974]
I am creating a custom checkbox for which I am using material-ui Checkbox using `customComponent : this.customMultiSelect`. in selectRowProp
I am using clickToSelect: true, and have a table of around 300 records.
The issue is coming in 3rd scenario
1. If I click on row , Checkbox gets activated/deactivated properly. Works well.
2. If I click on select All , All rows checkboxes gets activated/deactivated properly. Works well.
3. But If I Click on the row checkbox, it does not get checked or unchecked but instead throws an error on console. `Uncaught TypeError: Cannot read property 'id' of undefined` on `BootstrapTable.handleSelectRow`
I am not able to figure out  the issue as row object is not passed properly to the above function.
I m using chrome Version 52.0.2743.116 m (64-bit).
react-bootstrap-table: v3.0.0-beta.2

My code in custom component is 
```
return (<div>
          <Checkbox id = {'checkbox'+rowIndex} ref = { input=>{if(input) { input.indeterminate = props.indeterminate}}} key= {'checkbox'+rowIndex}   checked={checked} disabled = {disabled} onCheck ={(e) => onChange(e)}/>
      
      </div>)
```

**Comment by AllenFang** [comment]
sorry for lately reply, I'll check this out 👍

**Comment by AllenFang** [comment]
Fix on `v3.0.0-beta.7` https://github.com/AllenFang/react-bootstrap-table/releases/tag/v3.0.0-beta.7

**Comment by Ajaybhardwaj7** [comment]
Thanks Allen.
